specification/src/main/asciidoc/platform/Security.adoc

```diff
@@ -44,16 +44,16 @@ tamper-resistant record of security related events for the purpose of
 being able to evaluate the effectiveness of security policies and
 mechanisms.
 
-This chapter specifies how Java EE platform
+This chapter specifies how Jakarta EE platform
 requirements address security requirements, and identifies requirements
-that may be addressed by Java EE Product Providers. Finally, issues
+that may be addressed by Jakarta EE Product Providers. Finally, issues
 being considered for future versions of this specification are briefly
 mentioned in link:#a457[See Future
 Directions]”.
 
 === A Simple Example
 
-The security behavior of a Java EE environment
+The security behavior of a Jakarta EE environment
 may be better understood by examining what happens in a simple
 application with a web client, a JSP user interface, and enterprise bean
```

Comment:
Change JSP to either Jakarta Server Pages, or just go generic like "server pages" (similar to the generic reference to "enterrpise bean business logic".)

Reply:
Using JSP is fine, as long as you include the appropriate disclaimer at the beginning of the spec.
I suspect we're going to want to use "JSP" elsewhere in the spec as well.